package.json

```diff
@@ -33,6 +33,11 @@
   },
   "babel": {
     "env": {
+      "development": {
+        "plugins": [
+          "babel-plugin-styled-components"
```

Comment:
This puts readable class names in the DOM for development so that we can trace elements back to their components. Example:

![image](https://user-images.githubusercontent.com/7697924/35840606-58285f48-0ac5-11e8-9e2c-35f51693409d.png)